src/MIDebugEngine/AD7.Impl/AD7PendingBreakpoint.cs

```diff
@@ -146,6 +146,16 @@ public void ClearBoundBreakpoints()
             }
         }
 
+        public void SetError(AD7ErrorBreakpoint bpError, bool sendEvent = false)
```

Comment:
I had intended to do this and forgot to make the actual change, good catch. 

---
In reply to: [64832465](https://github.com/Microsoft/MIEngine/pull/346#discussion_r64832465) [](ancestors = 64832465)